src/containers/guest-home-page/reset-password/ResetPassword.jsx

```diff
@@ -21,6 +20,8 @@ import { confirmPassword, password } from '~/utils/validations/login'
 import { snackbarVariants } from '~/constants'
 import { styles } from './ResetPassword.styles'
 import imgSuccess from '~/assets/img/email-confirmation-modals/success-icon.svg'
+import AppButton from '~/components/app-button/AppButton'
```

Comment:
move it up to oher components

Reply:
Rearranged imports. While doing it also fixed some typos and small issues, for example error in console caused by `EmptyCooperationActivities` container